taskcluster/kinds/push-image/kind.yml

```diff
@@ -0,0 +1,157 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+
+loader: taskgraph.loader.transform:loader
+
+kind-dependencies:
+    - docker-image
+
+transforms:
+    - taskgraph.transforms.run:transforms
+    - taskgraph.transforms.task:transforms
+
+task-defaults:
+    worker-type: images
+    description: "{name} k8s image builder py{python_version}"
+    name: "{name}-python{python_version}"
+    worker:
+        taskcluster-proxy: true
+        docker-image: {in-tree: skopeo}
+        max-run-time: 1800
+        env:
+            # TODO: This should probably be set in a transform? either that or remove it because we don't really use it
+            VCS_HEAD_REPOSITORY: scriptworker-scripts
+            VCS_HEAD_REV: main
+            APP_VERSION: 1.0.0
```

Comment:
We should figure out the actual requirements wrt version.json

Reply:
push-image also needs to know whether to push to dev or production, which doesn't seem to be handled yet?

Reply:
I don't think it's required, but I also think it can be handy to see what sources the image was built from, so doesn't hurt to leave around imo.

Reply:
It should at least be set to the right properties now, matching what we already have (had?)